backend/logs_helper/log_service.py

```diff
@@ -0,0 +1,17 @@
+from django_redis import get_redis_connection
+
+
+class LogService:
+    @staticmethod
+    def remove_logs_on_logout(session_id):
+
+        if session_id:
+            # Get the Redis connection
+            r = get_redis_connection("default")
+
+            key_pattern = f"logs:{session_id}*"
```

Comment:
NIT: This key creation / generation can be encapsulated in a function so that the push / retrieve from redis can follow the same logic. It'll be easier to maintain in case we wish to change this key later